app/services/fees/commitments/minimum/create_service.rb

```diff
@@ -35,9 +35,6 @@ def call
             taxes_amount_cents: 0
           )
 
-          taxes_result = Fees::ApplyTaxesService.call(fee: new_fee)
```

Comment:
Are we not applying taxes on commitments anymore?

Reply:
I removed it since usually taxes are applied only on one place where we need to determine if it is related to tax provider or not. It worked fine, but since taxes were applied in minimum commitment service, tax application was skipped here: https://github.com/getlago/lago-api/blob/7141f81087734e5dca1332884d073b8cf3ae3453/app/services/invoices/compute_amounts_from_fees.rb#L14. With this change, as far as I see, the logic has not changed, just tax application is moved to the right service. Wdyt?